app/components/course-leaderboard.ts

```diff
@@ -127,6 +128,7 @@ export default class CourseLeaderboardComponent extends Component<Signature> {
       result.push(
         new CourseLeaderboardEntry({
           status: entriesForUser.isAny('status', 'evaluating') ? 'evaluating' : entryWithHighestCourseStage!.status,
+          completedStageSlugs: entryWithHighestCourseStage!.completedStageSlugs,
```

Comment:
```suggestion
          completedStageSlugs: entryWithMostStageCompletions!.completedStageSlugs,
```

("HighestCourseStage" doesn't make sense anymore in this context. The logic is already accurate since `entryWithHighestCourseStage` is computed using `completedStagesCount`)